packages/link/test/link.test.ts

```diff
@@ -18,7 +18,9 @@ describe('Link', () => {
     it('loads', async () => {
```

Comment:
This is exactly the test I wanted for the `download` attribute, but can we call it `loads [download]` (or similar) and have it be a second test, after what was there previously so that we test on either side of the presence of this attribute?